tests/wp-env/config/setup.sh

```diff
@@ -25,5 +21,13 @@ for id in $(wp user list --field=ID)
 do wp user meta add "$id" "announcements_user_counter" 999
 done
 
+wp --user=admin elementor library import-dir /var/www/html/elementor-templates
+
+wp cache flush
+wp rewrite flush --hard
+wp elementor flush-css
+
+wp post list --post_type=page
```

Comment:
I know that this function appears in the original code.
But do we really need it?

Reply:
We do not!
Done: https://github.com/elementor/elementor/pull/29345/commits/e67b47b51c5dc0d041bb3bb2418891596c005b3d